.meta

```diff
@@ -18,6 +18,7 @@
     "plugins/meta-project": "git@github.com:mateodelnorte/meta-project.git",
     "plugins/meta-template": "git@github.com:patrickleet/meta-template.git",
     "plugins/meta-yarn": "git@github.com:mateodelnorte/meta-yarn.git",
-    "templates/meta-plugin": "git@github.com:patrickleet/meta-template-meta-plugin.git"
+    "templates/meta-plugin": "git@github.com:patrickleet/meta-template-meta-plugin.git",
+    "templates/servicebus-microservice": "git@github.com:patrickleet/meta-template-servicebus-microservice.git"
```

Comment:
Weird, I had this comment, and it was stuck in a "ready to submit your review?" state.